docs/source/quickstart.mdx

```diff
@@ -1,66 +1,83 @@
-# Quick Start
-
-The quick start is intended for developers who are ready to dive in to the code, and see an end-to-end example of how they can integrate 🤗 Datasets into their model training workflow. For beginners who are looking for a gentler introduction, we recommend you begin with the [tutorials](./tutorial).
-
-In the quick start, you will walkthrough all the steps to fine-tune [BERT](https://huggingface.co/bert-base-cased) on a paraphrase classification task. Depending on the specific dataset you use, these steps may vary, but the general steps of how to load a dataset and process it are the same.
+# Quickstart
+
+This quickstart is intended for developers ready to dive into the code and see an end-to-end example of how to integrate 🤗 Datasets into their model training workflow. If you are a beginner or a non-developer, we recommend starting with our [tutorials](./tutorial), where you will get a more thorough introduction.
+
+Each dataset is unique, and depending on the task, some datasets may require additional steps to prepare the data. But you can always use 🤗 Datasets tools to load and process a dataset. Choose the type of dataset you want to work with, and let's get started!
+
+<div class="mt-4">
+   <div class="w-full flex flex-col space-y-4 md:space-y-0 md:grid md:grid-cols-3 md:gap-y-4 md:gap-x-5">
+      <a class="!no-underline border dark:border-gray-700 p-5 rounded-lg shadow hover:shadow-lg" href="/docs/datasets/quickstart#nlp"
+         ><div class="w-full text-center bg-gradient-to-br from-emerald-400 to-blue-500 rounded-lg py-1.5 font-semibold mb-5 text-white text-lg leading-relaxed">NLP</div>
+         <p class="text-gray-700">Tokenize a dataset and train a model to determine whether a pair of sentences have the same meaning.</p>
+      </a>
+      <a class="!no-underline border dark:border-gray-700 p-5 rounded-lg shadow hover:shadow-lg" href="/docs/datasets/quickstart#audio"
+         ><div class="w-full text-center bg-gradient-to-br from-violet-400 to-orange-500 rounded-lg py-1.5 font-semibold mb-5 text-white text-lg leading-relaxed">Audio</div>
+         <p class="text-gray-700">Resample an audio dataset and get it ready for a model to classify what type of banking issue a speaker is calling about.</p>
+      </a>
+      <a class="!no-underline border dark:border-gray-700 p-5 rounded-lg shadow hover:shadow-lg" href="/docs/datasets/quickstart#vision"
+         ><div class="w-full text-center bg-gradient-to-br from-cyan-400 to-pink-500 rounded-lg py-1.5 font-semibold mb-5 text-white text-lg leading-relaxed">Vision</div>
+         <p class="text-gray-700">Apply a data augmentation to an image dataset and get it ready for a model to diagnose disease in bean plants.</p>
+      </a>
+   </div>
+</div>
```

Comment:
The NLP and Audio labels don't render [here](https://moon-ci-docs.huggingface.co/docs/datasets/pr_4440/quickstart#quickstart) in my browser due to color selection (emerald and violet). Maybe [this](https://stackoverflow.com/questions/70704377/default-colors-given-in-tailwind-documentation-are-not-working) SO thread can help resolve the issue.

Reply:
Oh weird! They render normally in the `doc-builder` preview, and we should have support for these colors since we're using Tailwind v3. @mishig25 do I need to add emerald to the color palette (I see violet is already there)?

![Screen Shot 2022-06-07 at 9 41 23 AM](https://user-images.githubusercontent.com/59462357/172437251-2b43ebd8-305d-4cc2-85b6-92e1ffc10970.png)

Reply:
oh indeed, that is expected. 
everytime, a new tailwind class gets used, it needs to be added to [internal link](https://github.com/huggingface/moon-landing/blob/main/server/views/pages/DocPage/TailwindWhitelist.svelte)
because tailwind [removes everything that is not whitelisted](https://v2.tailwindcss.com/docs/optimizing-for-production#removing-unused-css)

Reply:
i love this screenshot but i would suggest to change the order of modalities (for instance Audio, Vision, NLP) to "break" the perception that NLP is the first class citizen (which it is... but we're trying to change that perception 😆)

cc @osanseviero @gary149